frontend/packages/pipelines-plugin/src/utils/pipeline-actions.tsx

```diff
@@ -119,11 +119,12 @@ const rerunPipeline: KebabAction = (
   kind: K8sKind,
   pipelineRun: PipelineRunKind,
   resources: any,
-  customData: RerunPipelineData = { labelKey: 'Start Last Run' },
+  customData: RerunPipelineData = {},
 ) => {
-  const { onComplete } = customData;
+  // t('pipelines-plugin~Start last run')
+  const { labelKey = 'pipelines-plugin~Start last run', onComplete } = customData;
 
-  const sharedProps = { labelKey: customData.labelKey, accessReview: {} };
+  const sharedProps = { labelKey, accessReview: {} };
```

Comment:
This is the functionality of that change. We didn't translate the labelKey when it was default.